HopsanGUI/Dialogs/LicenseDialog.cpp

```diff
@@ -44,7 +44,6 @@ LicenseDialog::LicenseDialog(QWidget *pParent) :
     resize(640,480);
     setWindowTitle(tr("Hopsan License"));
     setAttribute(Qt::WA_DeleteOnClose, true);
-    setPalette(QPalette(QColor("gray"), QColor("whitesmoke")));
```

Comment:
Should this line really be removed?

Reply:
Yes, I should have made a separate commit. It was required before for some reason to make the license text visible. But I have rewritten the dialog now and it looks strange if I force that style. Now it looks OK with both native and custom hopsan style

Reply:
I can rebase, lets try that before merge :)